README.md

```diff
@@ -9,8 +9,8 @@
 
 |           |   C                              |             J                    |
 |-----------|----------------------------------|----------------------------------|
-| Released  | [icu4c 68.1](./apidoc/released/icu4c) | [icu4j 69.1](./apidoc/released/icu4j) |
-| Dev       | [icu4c](./apidoc/dev/icu4c)      | [icu4j](./apidoc/dev/icu4j)      |
+| Released  | [icu4c 69.1](./apidoc/released/icu4c) | [icu4j 69.1](./apidoc/released/icu4j) |
+| Dev   | [icu4c](./apidoc/dev/icu4c)       | [icu4j](./apidoc/dev/icu4j)      |
```

Comment:
It would be nice to revert this line to exactly what it was before.

Reply:
Done.